sources/advanced/builtins/tokens.ts

```diff
@@ -0,0 +1,15 @@
+import {Command} from '../Command';
+import {Proxy}   from '../options';
+
+/**
+ * A command that prints the clipanion definitions.
+ */
+export class TokensCommand extends Command<any> {
```

Comment:
`TokensCommand` isn't exposed under `Builtins` at the moment.

Also, it should probably be registered in the demo.